CHANGELOG.asciidoc

```diff
@@ -30,6 +30,10 @@ This release also includes changes from <<release-3-6-XXX, 3.6.XXX>>.
 * Added warning on vertex property cardinality mismatch when reading GraphML.
 * Bumped to `ws` 8.x for `gremlin-javascript`.
 
+==== Improvements
+
+* TINKERPOP-2798 Add support for mid-traversal E()
```

Comment:
The JIRA entries to CHANGELOG are added on release. You should add an entry above. That entry is typically meant to be more descriptive than the JIRA title or even more specific as you might do multiple entries per JIRA item.

Reply:
updated in https://github.com/apache/tinkerpop/pull/1808/commits/41240b356c3cde71f673952d27ef061983402c9d